src/screens/profile/CreateAmenities/CreateAmenities.utils.ts

```diff
@@ -0,0 +1,154 @@
+import { IconName, IconSet } from 'src/types';
+
+export type AmenitiesErrorHandler = {
```

Comment:
usually handler is the name of a function or callback but not an object

Reply:
renamed to AmenitiesErrorType